src/api/git/getChangedFiles.ts

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs-extra';
+import git from 'isomorphic-git';
+
+import {
+  ABSENT,
+  FILEPATH,
+  HEAD,
+  STAGE,
+  UNCHANGED,
+  WORKDIR,
+} from './statusMatrix';
+
+export interface ChangedFile {
+  path: string;
+  deleted: boolean;
```

Comment:
Would it be useful to expose a tri-state `'added' | 'modified' | 'deleted'` here?

Reply:
Yeah I guess it would be handy to know the state. I think when I was doing it quickly to make it just "work" I only needed `deleted` but yeah I think it's worth exposing. I'll make it return exactly that ^ in a `state` variable.